### PR TITLE
[zh-cn]: Update translation of String.endsWith()

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

Same as the title.

### Motivation

Ditto.

### Additional details

None.

### Related issues and pull requests

#13975

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/string/endswith/index.md [hunk 1]
```diff
@@ -5,56 +5,47 @@ slug: Web/JavaScript/Reference/Global_Objects/String/endsWith
...
+endsWith(searchString, endPosition)
 ```
 
 ### 参数
 
 - `searchString`
-  - : 要搜索的子字符串。
-- `length` {{optional_inline}}
-  - : 作为 `str` 的长度。默认值为 `str.length`。
+  - : 要搜索的作为结尾的字符串，[不能是正则表达式](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/RegExp#正则表达式的特殊处理)。所有非正则表达式的值都会被[强制转换为字符串](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/String#字符串强制转换)，因此如果该参数被省略或传入 `undefined`，`endsWith()` 方法会在字符串中搜索`"undefined"`，这通常不是你想要的。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/endswith/index.md:22]:
```suggestion
  - : 要搜索的作为结尾的字符串，[不能是正则表达式](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/RegExp#正则表达式的特殊处理)。所有非正则表达式的值都会被[强制转换为字符串](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/String#字符串强制转换)，因此如果该参数被省略或传入 `undefined`，`endsWith()` 方法会在字符串中搜索 `"undefined"`，这通常不是你想要的。
```

File: files/zh-cn/web/javascript/reference/global_objects/string/endswith/index.md [hunk 2]
```diff
@@ -5,56 +5,47 @@ slug: Web/JavaScript/Reference/Global_Objects/String/endsWith
...
+- `endPosition` {{optional_inline}}
+  - : 预期找到 `searchString` 的末尾位置（即 `searchString` 最后一个字符的索引加 1 ）。默认为 `str.length`。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/endswith/index.md:24]:
```suggestion
  - : 预期找到 `searchString` 的末尾位置（即 `searchString` 最后一个字符的索引加 1）。默认为 `str.length`。
```

File: files/zh-cn/web/javascript/reference/global_objects/string/endswith/index.md [hunk 3]
```diff
@@ -5,56 +5,47 @@ slug: Web/JavaScript/Reference/Global_Objects/String/endsWith
...
 ```js
-var str = "To be, or not to be, that is the question.";
+const str = "大家好，我是练习时长两年半的个人练习生。";
 
-alert( str.endsWith("question.") );  // true
-alert( str.endsWith("to be") );      // false
-alert( str.endsWith("to be", 19) );  // true
+console.log(str.endsWith("练习生。")); // true
+console.log(str.endsWith("练习")); // false
+console.log(str.endsWith("练习", 8)); // true
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/endswith/index.md:48]: 不建议这么写，请保持中立

Comment (JasonLamv-t) [reply files/zh-cn/web/javascript/reference/global_objects/string/endswith/index.md:48]: 哈哈，皮一下很开心

File: files/zh-cn/web/javascript/reference/global_objects/string/endswith/index.md [hunk 4]
```diff
@@ -5,56 +5,47 @@ slug: Web/JavaScript/Reference/Global_Objects/String/endsWith
...
+这个方法可以让你确定一个字符串是否以另一个字符串结尾。该方法区分大小写。
 
 ## 示例
 
-### 使用 `endsWith()`
+### 使用 endsWith()
 
 ```js
-var str = "To be, or not to be, that is the question.";
+const str = "大家好，我是练习时长两年半的个人练习生。";
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/string/endswith/index.md:44]:
```suggestion
const str = "生存还是毁灭，这是一个问题。";
```

File: files/zh-cn/web/javascript/reference/global_objects/string/endswith/index.md (hunk 3 above)
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/string/endswith/index.md:48]:
```suggestion
console.log(str.endsWith("问题。")); // true
console.log(str.endsWith("毁灭")); // false
console.log(str.endsWith("毁灭", 6)); // true
```

**Review by yin1999 — APPROVED** [review]
Thanks.

**Pull request merged**